lib/Elastica/Cluster/Settings.php

```diff
@@ -155,13 +154,9 @@ public function setReadOnly($readOnly = true, $persistent = false)
     {
         $key = 'cluster.blocks.read_only';
 
-        if ($persistent) {
-            $response = $this->setPersistent($key, $readOnly);
-        } else {
-            $response = $this->setTransient($key, $readOnly);
-        }
-
-        return $response;
+        return $persistent
```

Comment:
In my opinion, the if block works better like this:

```php

if ($persistent) {
    return $this->setPersistent($key, $readOnly);
}

return $this->setTransient($key, $readOnly);
```

I find code that tries to follow Object Calisthenics to be nicer to read/maintain: http://williamdurand.fr/2013/06/03/object-calisthenics/